internal/stacks/stack_manager.go

```diff
@@ -360,6 +360,11 @@ func (s *StackManager) runStartupSequence(workingDir string, verbose bool, first
 	}
 
 	s.Log.Info("starting FireFly dependencies")
+
+	if err := docker.RunDockerComposeCommandWithRetry(workingDir, verbose, verbose, "pull"); err != nil {
+		return err
+	}
+
```

Comment:
We already do a pull (if `--no-pull` is not set) here:

https://github.com/hyperledger/firefly-cli/blob/e645f531923524119451d1b94d222fa87a497f24/internal/stacks/stack_manager.go#L487-L492

Should we just change that existing call to `RunDockerComposeCommandWithRetry`?

Reply:
Aha, this is a great point… but what it points at (which I totally missed) is the problem is the e2e test is doing a no-pull 🤦‍♂️

So 👍 to moving the retry on the existing pull, but it means this won’t actually fix the issue.

We might need to punt this to after your manifest work, and have a new behavior for the e2e test where it no long specifies no-pull, but instead uses a different (local) image name.

Reply:
I've made the change on this PR, and am up for working on the follow-on change to generate a special manifest in the e2e test code once the manifest work is ready